src/components/NegotiationPosts.vue

```diff
@@ -159,11 +165,12 @@ export default {
     async updateMessageStatus(inputMessageId, inputMessageText) {
       await this.markMessageAsRead({
         data: {
-          resourceId: this.negotiation.requests[0].resources[0].id,
+          resourceId: this.resourceId, //this.negotiation.requests[0].resources[0].id,
```

Comment:
if the comment is not necessary, please remove it

Reply:
Fixed in 338765dbdacd1bacd4779ccc2b2d75595643e489